### DISCUSSION TITLE
Linux installation

### DISCUSSION THREAD

**Discussion by perceptualrobots** [cyberbotics/wrestling#47] [Q&A] [answered]
Hello,

I am trying to install webots on linux (WSL ubuntu 20), following this link https://github.com/cyberbotics/webots/wiki/Linux-installation, which involves compiling.

Is this the correct link to follow?

At the make stage I get this error 

```
make: /mnt/c/packages/webots/scripts/get_git_info/get_git_info.sh: Command not found
make: *** [Makefile:150: docs] Error 127
```
Anyone know what the problem is?

Regards,
Rupert

**Comment by Jean-Eudes-le-retour** [comment, marked as answer]
The guides on the Webots repository wiki are there is you want to contribute to the Webots software. If you just want to use Webots on your computer, the preferred way is by following this guide: https://cyberbotics.com/doc/guide/installation-procedure

**Reply by perceptualrobots** [reply]
Thanks that install worked for me.

When I try to run the "webots" command I get this error, though I think this is an issue with WSL.

```
FATAL: Webots requires OpenGL 3.3 while only OpenGL 1.4 can be initialized.
Please check your GPU abilities and install the latest graphics drivers.
Please do also check that your graphics hardware meets the requirements specified in the User Guide.

```

**Reply by omichel** [reply]
Yes, we don't recommend that you use Webots from WSL. You should use it either on Windows or on Linux, but WSL is not a good option.
